NFe.Classes/Servicos/Evento/Informacoes/CreditoCombustivel/gConsumoComb.cs

```diff
@@ -0,0 +1,73 @@
+/*  Esta biblioteca é software livre; você pode redistribuí-la e/ou modificá-la */
+/* sob os termos da Licença Pública Geral Menor do GNU conforme publicada pela  */
+/* Free Software Foundation; tanto a versão 2.1 da Licença, ou (a seu critério) */
+/* qualquer versão posterior.                                                   */
+/*                                                                              */
+/*  Esta biblioteca é distribuída na expectativa de que seja útil, porém, SEM   */
+/* NENHUMA GARANTIA; nem mesmo a garantia implícita de COMERCIABILIDADE OU      */
+/* ADEQUAÇÃO A UMA FINALIDADE ESPECÍFICA. Consulte a Licença Pública Geral Menor*/
+/* do GNU para mais detalhes. (Arquivo LICENÇA.TXT ou LICENSE.TXT)              */
+/*                                                                              */
+/*  Você deve ter recebido uma cópia da Licença Pública Geral Menor do GNU junto*/
+/* com esta biblioteca; se não, escreva para a Free Software Foundation, Inc.,  */
+/* no endereço 59 Temple Street, Suite 330, Boston, MA 02111-1307 USA.          */
+/* Você também pode obter uma copia da licença em:                              */
+/* http://www.opensource.org/licenses/lgpl-license.php                          */
+/*                                                                              */
+/* Zeusdev Tecnologia LTDA ME - adenilton@zeusautomacao.com.br                  */
+/* http://www.zeusautomacao.com.br/                                             */
+/* Rua Comendador Francisco josé da Cunha, 111 - Itabaiana - SE - 49500-000     */
+/********************************************************************************/
+
+using System.Xml.Serialization;
+
+namespace NFe.Classes.Servicos.Evento.Informacoes.CreditoCombustivel
+{
+    public class gConsumoComb
+    {
+        private decimal _vIBS;
+        private decimal _vCBS;
+        
+        /// <summary>
+        ///     P24 - Corresponde ao atributo “nItem” do elemento “det” do documento referenciado.
+        /// </summary>
+        [XmlAttribute]
+        public int nitem { get; set; }
+        
+        /// <summary>
+        ///     P25 - Valor do IBS relativo ao consumo de combustível na nota de aquisição
+        /// </summary>
+        public decimal vIBS
+        {
+            get => _vIBS; 
+            set => _vIBS = value.Arredondar(2); 
+        }
+        
+        /// <summary>
+        ///     P26 - Valor da CBS relativo ao consumo de combustível na nota de aquisição
+        /// </summary>
+        public decimal vCBS
+        {
+            get => _vCBS; 
+            set => _vCBS = value.Arredondar(2); 
+        }
+        
+        /// <summary>
+        ///     P27 - Informações de quantidade por item
+        /// </summary>
+        [XmlElement(ElementName = "gControleEstoque")]
+        public gControleEstoqueCredComb gControleEstoqueCredComb { get; set; }
```

Comment:
Manter nome da propriedade a mesma do que será serializado no XML, para melhor identificar no momento de preencher:
```suggestion
        public gControleEstoqueCredComb gControleEstoque { get; set; }
```

Reply:
Ajustado